openreview/journal/templates.py

```diff
@@ -283,4 +283,23 @@
 A new submission has been received for {short_name}.
 
 To view the submission, click here: https://openreview.net/forum?id={submission_id}
+'''
+
+author_new_submission_email_template = '''Hi {{{{fullname}}}},
+
+Your submission to {short_name} has been received.
+
+Submission Number: {submission_number}
+
+Title: {submission_title}
+
+To view the submission, click here: https://openreview.net/forum?id={submission_id}
+'''
+
+author_official_recommendation_starts_email_template = '''Hi {{{{fullname}}}},
+
+The discussion period has ended and the reviews will submit their recommendation, upon your AE then will enter their final recommendation
```

Comment:
I know this is the template the PCs provided, but maybe this is more clear:
```suggestion
The discussion period has ended and the reviewers will submit their recommendations, after which the AE will enter their final recommendation.
```